app/models/device.rb

```diff
@@ -61,7 +61,7 @@ def recreate_ios_app!
     app = Rpush::Apns2::App.new
     app.name = app_bundle
     sanitized_pem = ApplicationConfig["RPUSH_IOS_PEM"].to_s.gsub("\\n", "\n")
-    app.certificate = Base64.decode64(sanitized_pem)
+    app.certificate = sanitized_pem
```

Comment:
This not breaking any tests worries me :D

Reply:
This is a port from @fdoxyz's upcoming branch. What should I do here @fdoxyz?

Reply:
Yup, this will be extracted in that follow up PR you mention @joshpuetz. I'm totally okay with merging main and resolving any conflicts in my PR separately, because this fixes the way it currently works. I already have tests around updating the credentials, also tests around the admin dashboard providing feedback when/if this certificate is valid/invalid. I can make sure to add a few more if possible though.

All this to say LGTM despite knowing it could be more robust 🙂

Reply:
Cool, I'll keep this as is since tests will be coming in with @fdoxyz's PR